tests/alternative-deps/WORKSPACE

```diff
@@ -52,10 +52,22 @@ gazelle_cabal_dependencies(
 
 stack_snapshot(
     name = "stackage-b",
+    components = {
+        "tasty-discover": [
+            "lib",
+            "exe:tasty-discover",
+        ],
+    },
     packages = [
         "aeson",  # keep
+        "base",
+        "inspection-testing",
         "path",  # keep
         "path-io",  #keep
+        "tasty",
+        "tasty-discover",
+        "tasty-hunit",
+        "void",
```

Comment:
These changes look unintended, if `gazelle-update-repo` stills generates this output.